index.bs

```diff
@@ -558,8 +564,8 @@ authorizing an authenticator with which to complete the operation.
 
     The <dfn>type</dfn> member specifies the type of credential to be created.
 
-    The <dfn>algorithm</dfn> member specifies the cryptographic algorithm with which the newly generated credential will be
-    used.
+    The <dfn>algorithm</dfn> member specifies the cryptographic key generation algorithm parameters with which the credential 
+    key pair MUST be generated.
```

Comment:
This changes the meaning of the sentence, and in a potentially inaccurate way. Two problems here - changing "cryptographic algorithm" to "key generation algorithm" and changing "algorithm" to "algorithm parameters".

The algorithm ID is the signature algorithm ID. It is not the key generation algorithm ID, which can be different (e.g. there are multiple algorithms to generate an RSA key, but we would not be specifying which one of those to use, we'd just be saying RSA).

Also, the algorithm may or may not have parameters (e.g. with RSA one might specify a modulus length but maybe not with other algorithms). In this case I think you mean "parameters" as in parameters to the WebAuthn API but due to the overloaded nature of the term this would not be the obvious reading I think.